patch/llpcPatchInOutImportExport.h

```diff
@@ -344,6 +346,8 @@ class PatchInOutImportExport:
     std::vector<llvm::CallInst*> m_importCalls; // List of "call" instructions to import inputs
     std::vector<llvm::CallInst*> m_exportCalls; // List of "call" instructions to export outputs
     PipelineState*          m_pPipelineState = nullptr; // Pipeline state from PipelineStateWrapper pass
+
+    std::set<uint32_t>       m_vsExpLoc; // The locations that already have an export instruction for the vertex shader.
```

Comment:
m_vsExpLoc -> m_expLocSet or m_expLocs
No need to add "vs" prefix because the vs is hardware VS and in this pass, we often denote "vs" as API VS.